src/dependencies.py

```
@@ -89,6 +130,153 @@ def get_flows_service(services: dict = Depends(get_services)):
+
+    When ``OPENRAG_RBAC_ENFORCE=false`` the check is skipped and the
+    authenticated user is returned unconditionally. The startup event
+    in ``src/main.py`` refuses to boot when this flag is combined with
+    a ``saas`` or ``on_prem`` run mode, so the bypass cannot silently
+    land in production.
+    """
+    from services.rbac_service import is_rbac_enforced
+
+    async def _dep(
+        request: Request,
+        user: User = Depends(get_current_user),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)